docs/docs/plugins.md

```diff
@@ -2,42 +2,50 @@
 title: Plugins
 ---
 
-Plugins are Node.js packages that implement Gatsby APIs. They enable you to
-easily solve common website build problems e.g. setup Sass, add markdown
-support, process images, etc.
+One of the best ways to add functionality to Gatsby is through our plugin system. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
 
-For larger / complex sites, they let you modularize your site customizations
-into site-specific plugins.
+Of the many possibilities, plugins can:
 
-Gatsby has a large and growing set of plugins. To search/browse official and
-community plugins and their documentation, visit the [Plugin Library](/plugins/).
+- add external data or content (e.g. your CMS, static files, a REST API) to your Gatsby GraphQL data
```

Comment:
Nice, this does seem to belong here rather than in /plugin-authoring/. These docs have needed to be re-organized for a long time! What a relief!